resources/kcp/charts/kyma-environment-broker/templates/trial-cleanup-job.yaml

```diff
@@ -91,7 +91,7 @@ spec:
                 - name: APP_BROKER_SCOPE
                   value: {{.Values.kebClient.scope}}
               command:
-                - "/bin/trialcleanup"
+                - "/app/main"
```

Comment:
It will break the job.

Reply:
Your changes should not affect jobs other than environment cleanup (according to what you said).